dev/docker-compose.yml

```diff
@@ -142,6 +142,49 @@ services:
     profiles:
       - worker
 
+  monitor:
+    build: ../monitor/parent
+    container_name: zf_monitor
+    restart: unless-stopped
+    ports:
+      - 127.0.0.1:8004:19999
+    cap_add:
+      - SYS_PTRACE
+    security_opt:
+      - apparmor:unconfined
+    environment:
+      - ZIMFARM_API_URL=http://backend:80/v2
+      - ZIMFARM_USERNAME=admin
+      - ZIMFARM_PASSWORD=admin
+      - NETDATA_HOSTNAME=monitor
+    volumes:
+      - netdataconfig:/etc/netdata
+      - netdatalib:/var/lib/netdata
+      - netdatacache:/var/cache/netdata
+      - /etc/passwd:/host/etc/passwd:ro
+      - /etc/group:/host/etc/group:ro
+      - /etc/localtime:/etc/localtime:ro
+      - /proc:/host/proc:ro
+      - /sys:/host/sys:ro
+      - /etc/os-release:/host/etc/os-release:ro
+      - /var/log:/host/var/log:ro
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    depends_on:
+      backend:
+        condition: service_healthy
+    profiles:
+      - worker
+
+  task_monitor:
```

Comment:
Add comment explaining this is just meant to build the child monitor image locally so that it is available to be used by the task manager

Reply:
+ rename container, image and container name to `monitor_child` and `zf_monitor_child` for clarity